test/karma/karma.conf.js

```diff
@@ -74,6 +74,7 @@ module.exports = function(config) {
         // level of logging
         // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
         logLevel: config.LOG_INFO,
+        logLevel: config.LOG_DEBUG,
```

Comment:
Yea it failed the circleCI build with the duplicate log levels, can just stick it back to INFO